package.json

```diff
@@ -10,7 +10,8 @@
     "watch": "tsc-watch --onSuccess \"npm start\"",
     "test": "jest --colors --verbose --forceExit",
     "start": "node ./dist/src/examples/poi-crosschecker/poi-crosschecker.js",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "test.integration": "bash src/examples/poi-crosschecker/tests/integration/scripts/compare-attestations.sh"
```

Comment:
I'm wondering if it would be nice to also add this command a cronjob of some sorts in docker compose? can be for a later discussion!:)

Reply:
that's a great idea! I was thinking of a way to automate that, for instance as you said to run the script automatically after the container has been running for a bit, as a way to sanity check that it's doing its job

Reply:
The only thing there was that I wasn't sure how to start it once we know the db is not empty, but then again maybe we could just poll every couple of seconds 🤔

Reply:
Hmm either should work? i think it might make sense to ping every couple blocks - if the operator is acting normal then it should expect to have empty results for POI divergence whether the db is filled or not. Then for malicious indexer we want to explicitly handle db being empty?